Resources/Prototypes/Entities/Structures/Power/smes.yml

```
@@ -113,3 +116,48 @@
   components:
   - type: Battery
     startingCharge: 0
+
+- type: entity
+  parent: BaseSMES
+  id: SMESAdvanced
+  suffix: Advanced, 16MJ
+  name: advanced SMES
+  description: An even-higher-capacity superconducting magnetic energy storage (SMES) unit.
+  components:
+  - type: Sprite
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)